blueprints/component-addon/index.js

```diff
@@ -4,9 +4,8 @@ const path = require('path');
 const stringUtil = require('ember-cli-string-utils');
 const getPathOption = require('ember-cli-get-component-path-option');
 const normalizeEntityName = require('ember-cli-normalize-entity-name');
-const useEditionDetector = require('../edition-detector');
```

Comment:
I think this file (`blueprints/edition-detector.js`) still needs to be deleted, right?

Reply:
Good catch